main.py

```diff
@@ -111,11 +112,34 @@ def rat_cache(self) -> object:
         return self._rat_cache
 
     @property
-    def database_mgr(self) -> object:
+    def fact_mgr(self) -> FactManager:
```

Comment:
Any particular reason for the abbreviation? I see that's how it used to be written, but it doesn't need to stay that way.

Reply:
i would far prefer `fact_manager`.

Reply:
This has been fixed.